src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -13,6 +13,8 @@
         <dd i18n>{{ detailsCardData.orchestrator || 'Orchestrator is not available' }}</dd>
         <dt>Ceph version</dt>
         <dd>{{ detailsCardData.cephVersion }}</dd>
+        <dt>API Documentation</dt>
+        <dd><a routerLink="/api-docs">{{ getAPIDocRoute() }}</a></dd>
```

Comment:
Its not a good idea to [bind methods to the templates](https://levelup.gitconnected.com/angular-dont-use-functions-in-templates-f33d67db18da) because of the changeDetection in angular. This method would gets executed a lot of times within a minute and keeps on doing that. You can test it out by putting a `console.log("fun")` in the function and see the fun going on in the browser dev tools. 

Instead you can define `origin = window.location.origin` inside the class and use `origin` in the html template

Also we could add `target="_blank"` to make it open in a separate tab.
```suggestion
        <dd><a routerLink="/api-docs"
               target="_blank">{{ origin }}/api-docs</a></dd>
```

Also what about an icon to indicate its opening in a separate window?
```
<a routerLink="/api-docs"
   target="_blank">{{ origin }}/api-docs
  <i class="fa fa-external-link"></i>
</a>
 ```